src/components/header/Header.js

```diff
@@ -187,33 +179,76 @@ export function Header({
-                        name: m,
-                        prefix: "header-link",
+
+                  <Link
+                    href="/inspiration/artikler"
+                    dataCy="header-link-article"
+                  >
+                    <Text type="text3">
+                      {Translate({
+                        context: "facets",
+                        label: `label-article`,
                       })}
-                      onClick={() => {
-                        filters.setQuery({ include: { workTypes: [m] } });
-                      }}
-                      onKeyDown={(e) => {
-                        if (e.key === "Enter" || e.keyCode === 13) {
-                          filters.setQuery({ include: { workTypes: [m] } });
-                        }
-                      }}
-                    >
-                      <Text type="text3">
-                        {Translate({
-                          context: "facets",
-                          label: `label-${m}`,
-                        })}
-                      </Text>
-                    </Link>
-                  ))}
+                    </Text>
+                  </Link>
+
+                  <Link href="/inspiration/spil" dataCy="header-link-game">
+                    <Text type="text3">
+                      {Translate({
+                        context: "facets",
+                        label: `label-game`,
+                      })}
+                    </Text>
+                  </Link>
+
+                  <Link
+                    href="/inspiration/boeger"
+                    dataCy="header-link-literature"
+                  >
+                    <Text type="text3">
+                      {Translate({
+                        context: "facets",
+                        label: `label-literature`,
+                      })}
+                    </Text>
+                  </Link>
+
+                  <Link href="/inspiration/film" dataCy="header-link-movie">
+                    <Text type="text3">
+                      {Translate({
+                        context: "facets",
+                        label: `label-movie`,
+                      })}
+                    </Text>
+                  </Link>
+
+                  <Link href="/inspiration/musik" dataCy="header-link-music">
+                    <Text type="text3">
+                      {Translate({
+                        context: "facets",
+                        label: `label-music`,
+                      })}
+                    </Text>
+                  </Link>
+
+                  <Link
+                    href="/inspiration/noder"
+                    dataCy="header-link-sheetmusic"
+                  >
+                    <Text type="text3">
+                      {Translate({
+                        context: "facets",
+                        label: `label-sheetmusic`,
+                      })}
+                    </Text>
+                  </Link>
```

Comment:
For at undgå code-duplikation, kan du lave et array af arrays/objekter og iterere over den

Reply:
Ja det går nok :)